python/ribasim_testmodels/ribasim_testmodels/invalid.py

```diff
@@ -215,7 +220,7 @@ def invalid_priorities_model() -> Model:
     )
     model.edge.add(model.tabulated_rating_curve[2], model.basin[3])
     model.edge.add(model.basin[3], model.user_demand[4])
-    model.edge.add(model.basin[3], model.level_demand[6])
```

Comment:
Switched the position of basin[3] and level_demand[6] because it was a mistake

Reply:
But this is an invalid model, so that might've been on purpose?

Reply:
No, I didn't do it on purpose. I only wanted to use this model to test the error of priority. But now it gave me two errors because I accidently connected node types that is not valid. So I corrected